en/docs/graphql-api.md

```diff
@@ -0,0 +1,86 @@
+
+    A GraphQL API can retrieve all these details via one query similar to the sample given below:
+
+    ```
+    query {
+        customer (id: "SIL182882762") {
+            contactInfo {
+                phoneNo
+            }
+            balances(current)
+        }
+    }
+    ```
+    In this scenario, a single request by a REST API under-fetches data compared to a single request by a GraphQL API.
+
+- In some scenarios, a REST API can over-fetch data. In the same example given above, a request sent by REST API to the `/customers/<id>/contactInfo` may result in fetching all the contact information in the array such as the phone number, work phone number, address, email, etc., whereas the application only needs the phone number.
+
+    A GraphQL allows you to avoid this by writing queries with specific data requirements.
+
+By minimizing the number of requests it sends for a given purpose and avoiding over-fetching GraphQL API consumes a smaller amount of resources (such as the internet bandwidth, system memory, etc.). GraphQL APIs are therefore more economical. Therefore it is beneficial to consume GraphQL APIs for low-energy devices such as mobile devices compared to consuming REST APIs.
+
+
+## Develop
+
+To create a GraphQL you need to add a GraphQL component.
+
+![Add GraphQL component](assets/img/graphql/add-graphql-component.png){.cInlineImage-full}
+
+Choreo comes with the Web Editor which is an IDE designed based on Visual Studio Code for developers to implement API functionality. At present, developers can design GraphQL APIs in the Web Editor by writing code, and test the functionality via cURL commands before committing the design to a private/Choreo-managed GitHub repository. Once you commit the design, the GraphQL API is available in the Choreo Console where you can deploy, test, and observe it.
+
+![Implement graphql API](assets/img/graphql/implement-graphql-api.png){.cInlineImage-full}
+
+!!! info
+    The current preview does not support editing the GraphQL API implementation in the low-code mode.
+
+## Lifecycle
+
+Once you implement a GraphQL API, it needs to be built, tested, and eventually put into production. Choreo executes a CI/CD pipeline to manage the API lifecycle. Choreo stores the source code of APIs in a GitHub repository for the user account. This GitHub repository can be a Choreo-managed repository or a GitHub repository that you own.  It also provides the capability for developers to connect their own GitHub repositories containing the source code of their APIs. The CI/CD pipeline in Choreo takes an API through the complete process, starting from code checkout to compile, build, test, and finally deploy.
```

Comment:
Suggestion:
"Once you implement a GraphQL API, you can build, test, and eventually put it into production."